azure-iot-provisioning-devicessdk/azure/iot/provisioning/devicessdk/transport/device_provisioning_constant.py

```diff
@@ -0,0 +1,23 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+"""
+This module contains constants related to the transport package.
+"""
+
+USER_AGENT = "azure-iot-provisioning-device" + "/" + "1.5.3"
```

Comment:
>azure-iot-provisioning-device" + "/" + "1.5.3" [](start = 14, length = 46)

probably want this to identify that it's the python preview SDK and reflect a proper version (right now this will look like node). I would suggest "<pip_package_name>/<pip_package_version>"

Reply:
I have changed to the format. however our package name does not contain the word "preview" ,

---
In reply to: [265324593](https://github.com/Azure/azure-iot-sdk-python-preview/pull/59#discussion_r265324593) [](ancestors = 265324593)